### ISSUE TITLE
Throw error or print error message when no toolboxes are deployed

### ISSUE THREAD

**Issue by spitschan** [ToolboxHub/ToolboxToolbox#13]
When `tbUse` is called with a non-existing toolbox configuration, the misleading message `Looks good: all toolboxes deployed OK.` is displayed, making it very hard to debug. Maybe it would make sense to throw an error, or display some error message, when a called-for toolbox configuration does not exist?

```
>> tbUse('ThisToolboxConfigDoesNotExist')
Updating "ToolboxRegistry".
Adding "ToolboxToolbox" to path at "/Users/mspits/Documents/MATLAB/Toolboxes/ToolboxToolbox".
Looks good: all toolboxes deployed OK.

ans = 

0x0 struct array with fields:

    flavor
    hook
    importance
    localHookTemplate
    name
    pathPlacement
    subfolder
    toolboxRoot
    type
    update
    url
    path
    status
    message
    operation
    command
    strategy
    isOk
```

**Comment by benjamin-heasly** [comment]
Absolutely.

**Comment by benjamin-heasly** [comment]
This was addressed in 04906b93a4484afc29b940af36b2c1a779c80345.  I forgot to update this issue at the time!
